src/main/scala/io/viash/config/Config.scala

```diff
@@ -87,7 +89,30 @@ case class Config(
   engines: List[Engine] = Nil,
 
   @undocumented
-  info: Option[Info] = None
+  info: Option[Info] = None,
+
+  @description("The project config content used during build.")
+  @since("Viash 0.9.0")
+  @undocumented
+  project_config: Option[ProjectConfig] = None,
+
+  @description("The keywords of the components.")
+  @example("keywords: [ bioinformatics, genomics ]", "yaml")
+  @default("Empty")
+  @since("Viash 0.9.0")
+  keywords: List[String] = Nil,
+
+  @description("The license of the project.")
+  @example("license: MIT", "yaml")
+  @default("Empty")
+  @since("Viash 0.9.0")
+  license: Option[String] = None,
+
+  @description("The organization of the project.")
+  @example("organization: viash-io", "yaml")
+  @default("Empty")
+  @since("Viash 0.9.0")
+  organization: Option[String] = None,
```

Comment:
I made a mistake in the original issue -- this should be in the `.functionality` (since `name`, `description`, ... is there)

Reply:
Fixed ^_^